.github/workflows/merge-flow.yaml

```diff
@@ -182,6 +182,12 @@ jobs:
             fi
             go mod tidy
             go mod vendor
+            ${{ inputs.assets-cmd }}
+            if [ -f scripts/rh-manifest.sh ]; then
+              bash scripts/rh-manifest.sh
+              git add rh-manifest.txt
+              git diff --cached --exit-code || git commit -s -m "[bot] update rh-manifest.txt"
```

Comment:
so basically the idea here is to avoid committing if there's no diff to generate, isn't it?

Reply:
yep, the commit only needs to me made if `rh-manifest.txt` actually changed. But note that this is only a descruption that gets posted to a PR in some cases. So a human actually executes the steps.